components/sections/docs/docs-content.tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsx jsx */
+
+import { jsx } from "theme-ui";
+
+export const Heading = ({ children }) => {
+  return (
+    <h1
+      sx={{
+        fontSize: ["40px", "48px"],
+        mb: "32px",
+        color: "docs.text",
+        fontWeight: "bold",
+        fontFamily: "special",
+      }}>
+      {children}
+    </h1>
+  );
+};
+
+export const SubHeading = ({ children }) => {
+  return (
+    <h1
```

Comment:
We are hardcoding the `h1` here, which is incorrect

Reply:
The editor should be able to put up to `h3`. The `h3` is the same but a bit smaller, and the margins tighter